src/ott/solvers/linear/sinkhorn.py

```diff
@@ -103,19 +136,19 @@ def solution_error(
   # and h is either f or g. Note this is equal to z if rho_z → inf, which
   # is the case when tau_z → 1.0
   if lse_mode:
```

Comment:
it seems I forgot to add `only_check_b` in the list of args above when I introduced it in a previous PR. Could you correct this?